benchmarks/src/main/java/com/google/samples/apps/nowinandroid/GeneralActions.kt

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2023 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.samples.apps.nowinandroid
+
+import android.Manifest.permission
+import android.os.Build.VERSION.SDK_INT
+import android.os.Build.VERSION_CODES.TIRAMISU
+import androidx.benchmark.macro.MacrobenchmarkScope
+
+/**
+ * Because the app under test is different from the one running the instrumentation test,
+ * the permission has to be granted manually by either:
+ * 
```

Comment:
Sorry for the extraneous space 🙈

```suggestion
 *
```